test/Microsoft.TestPlatform.Acceptance.IntegrationTests/DifferentTestFrameworkSimpleTests.cs

```diff
@@ -70,6 +70,8 @@ public void CPPRunAllTestExecutionPlatformx64Net(RunnerInfo runnerInfo)
     [TestMethod]
     [TestCategory("Windows-Review")]
     [NetFullTargetFrameworkDataSource]
+    // After the bump due to the new vesion it doesn't fine anymore a correct Microsoft.VisualStudio.Telemetry version, new version are taking newer versions of it and the test if for the legacy platform.
```

Comment:
I would remove this comment as you are provided a good ignore reason.

```suggestion
```

Reply:
I'll do in the followup